src/main/java/com/ubb/zenith/model/User.java

```diff
@@ -29,6 +29,7 @@ public class User {
     private String username;
 
     @NotBlank(message = "Email cannot be empty")
+    @Email(message = "Email should be valid")
```

Comment:
We had some problems with @Email. Let's keep it simple and use this annotation other time :)